renderite/serialization.zig

```diff
@@ -0,0 +1,79 @@
+const std = @import("std");
+const builtin = @import("builtin");
+
+const Reader = std.io.Reader;
+const Writer = std.io.Writer;
+
+const endian = builtin.cpu.arch.endian();
+
+pub const IpcDeserializer = struct {
+    reader: *Reader,
+
+    pub fn init(reader: *Reader) IpcDeserializer {
+        return IpcDeserializer{ .reader = reader };
```

Comment:
`T{}` syntax is pseudo-deprecated and can introduce unneeded copies, let's avoid it.

```suggestion
        return .{ .reader = reader };
```